frontend/styles/var/mixins.scss

```diff
@@ -36,3 +36,17 @@ $max-width: 2560px;
   margin: 0;
   padding: $pad-xxlarge 0 54px 0;
 }
+
+@mixin grey-badge {
+  background-color: $ui-fleet-black-25;
+  color: $core-white;
+  font-size: $xx-small;
+  font-weight: 700;
+  padding: 0 4px;
+  border-radius: 4px;
+  position: relative;
+  top: -2px;
+  min-width: 95px;
+  max-height: 19px;
```

Comment:
also `min-width` and `max-height` have no effect currently, with `display: inline`

Reply:
but if this mixin is going to be used elsewhere it might make a difference there, so 🤷🏻‍♂️

Reply:
sgtm, I just realized I should add this to the team members page too, so not ready for review until that's updated!